website/src/components/BlogsSlider/index.tsx

```diff
@@ -1,7 +1,5 @@
 import React from "react";
-import Slider from "react-slick";
-import "slick-carousel/slick/slick.css";
-import "slick-carousel/slick/slick-theme.css";
+import Carousel from "../Carousel";
```

Comment:
@rakeshPRaghu will this also resolve the two extra arrow button that appears at some places when using the `react-slick `

Reply:
added the style for that in the last commit